code/game/objects/structures/flora.dm

```diff
@@ -78,7 +78,31 @@
 	desc = "Oh, no. Not again"
 	icon = 'icons/obj/plants.dmi'
 	icon_state = "plant-26"
-
+	layer = FLY_LAYER

```

Comment:
This is the same layer used by chairs. I wanted to be internally consistent with other objects on the station.